### PR TITLE
SHOW REGIONS

### LINKED ISSUE #9095: sql - Implement SHOW REGIONS
PR: https://github.com/cockroachdb/cockroach/pull/55831

From release notes:
> Implemented `SHOW REGIONS`, which returns all of regions available in a cluster. [#55831][#55831] {% comment %}doc{% endcomment %}

### LINKED ISSUE #9113: sql: rework SHOW REGIONS to SHOW REGIONS FROM CLUSTER
PR: https://github.com/cockroachdb/cockroach/pull/56627

From release notes:
> `SHOW REGIONS` functionality is now deferred to `SHOW REGIONS FROM CLUSTER`. [#56627][#56627] {% comment %}doc{% endcomment %}

### LINKED ISSUE #9124: delegate: implement SHOW REGION FROM DATABASE
PR: https://github.com/cockroachdb/cockroach/pull/57106

From release notes:
> Implemented `SHOW REGION FROM DATABASE` and `SHOW REGION FROM DATABASE db`, which shows all regions for the given database, as well as whether that region is the primary region. [#57106][#57106] {% comment %}doc{% endcomment %}

Related to https://cockroachlabs.atlassian.net/browse/CRDB-1504.

### LINKED ISSUE #9308: Add docs for SHOW REGIONS .. statements
+ `SHOW REGIONS FROM CLUSTER`
+ `SHOW REGIONS FROM DATABASE 'foo'`
+ `SHOW REGIONS FROM ALL DATABASES`

### LINKED ISSUE #9564: sql: implement SHOW REGIONS FROM ALL DATABASES
PR: https://github.com/cockroachdb/cockroach/pull/57500

From release notes:
> Introduced `SHOW REGIONS FROM ALL DATABASES`, which shows region metadata for each database in the table. [#57500][#57500] {% comment %}doc{% endcomment %}

### LINKED ISSUE #9600: delegate: implement SHOW REGIONS
PR: https://github.com/cockroachdb/cockroach/pull/57618

From release notes:
> Implemented `SHOW REGIONS`, which shows a list of regions along with the databases associated with them. [#57618][#57618] {% comment %}doc{% endcomment %}

### LINKED ISSUE #10306: release-21.1: delegate: order AZs in SHOW REGIONS commands
PR: https://github.com/cockroachdb/cockroach/pull/62619

From release notes:
> Availability zones are now ordered when using the `SHOW REGIONS` set of commands. [#62619][#62619] {% comment %}doc{% endcomment %}

### PR DESCRIPTION
**Pull request by ericharmeling**
Fixes #9564.
Fixes #9113.
Fixes #9095.
Fixes #9600.
Fixes #9308.
Fixes #9124.
Fixes #10306.

Related to https://github.com/cockroachdb/cockroach/pull/64103.

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -104,6 +104,7 @@ GEM
 PLATFORMS
   ruby
   universal-darwin-19
+  universal-darwin-20
```
Comment (rmloveland) [inline Gemfile.lock:107]: I don't think we are supposed to change this file in the repo? Or am I misunderstanding?

File: _includes/v21.1/sql/generated/diagrams/show_regions.html [hunk 2]
```diff
@@ -0,0 +1,29 @@
+<div><svg width="683" height="189">
```
Comment (rmloveland) [inline _includes/v21.1/sql/generated/diagrams/show_regions.html:1]: BEEE YOO TEE FULL

**Review by rmloveland — APPROVED** [review]
This LGTM, I suggest having @otan from the eng world take a peek to verify the perms n stuff

Thanks Eric!  YOU ARE THE BEST

**Review by otan — APPROVED** [review]

File: Gemfile.lock (hunk 1 above)
Comment (ericharmeling) [reply Gemfile.lock:107]: I kind of thought I'd just sneak this into this PR hehe. 

@yingzhuchin 
This file updated after I was prompted yesterday to run `bundle install` on my machine before I could do a local build. IIUC, Big Sur runs on Darwin 20 (and I'm running Big Sur), while Catalina runs on Darwin 19, so I assume this is necessary for the build to work on Big Sur? IDK though. Just conjecture.

Comment (yingzhuchin) [reply Gemfile.lock:107]: I'm on Big Sur as well and `bundle install` doesn't add new platforms to my Gemfile.lock. What version of bundler are you running?

Comment (ericharmeling) [reply Gemfile.lock:107]: Hmm...

Bundler version 2.2.1

I'll just remove the change from this PR so I can merge.

**Pull request merged**